lib/request.js

```diff
@@ -131,6 +126,7 @@ const ajax = ({ method, url, headers, body = null }, callback) => {
 
   xhr.onerror = (e) => {
     console.log(e);
+    alert('There was a problem with that request. Please try again.');
```

Comment:
@joeosburn let's put the alert after the `swap.request = false;`

Reply:
ok moved